src/Constants/RecurrenceTypes.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * This file contains the different recurrence types.
+ *
+ * Copyright (C) 2021 - today Unzer E-Com GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @link  https://docs.unzer.com/
+ *
+ * @author  David Owusu <development@unzer.com>
+ *
+ * @package  UnzerSDK\Constants
+ */
+namespace UnzerSDK\Constants;
+
+class RecurrenceTypes
+{
+    /** @var string  Recurring with a defined interval and a defined amount.*/
+    public const SCHEDULED = 'scheduled';
+
+    /** @var string  Recurring with a undefined interval and/or an undefined amount.*/
+    public const UNSCHEDULED = 'unscheduled';
+
+    /**
+     * @var string Should only be used for customer initiated transactions.
+     *             The following transaction can’t be MIT (Merchant initiated transaction).
```

Comment:
I would keep it simple then and and just use first sentence. (See change)

Reply:
Ok, but I am also thinking about removing the CIT/MIT stuff completely, since it is of no concern to the merchant.
CIT/MIT is just between PAPI and core the merchant does not know about it.